serialization/dotnet/json/src/JsonParseNode.cs

```diff
@@ -49,7 +49,13 @@ public JsonParseNode(JsonElement node)
         /// Get the float value from the json node
         /// </summary>
         /// <returns>A float value</returns>
-        public decimal? GetFloatValue() => _jsonNode.GetDecimal();
+        public float? GetFloatValue() => (float)_jsonNode.GetDecimal();
```

Comment:
Simply curious. Any specific reason for changing the type from `decimal` to `float` from the abstractions side?
Also, we could change this to simply be
```suggestion
        public float? GetFloatValue() => _jsonNode.GetSingle();
```

Reply:
ah this naming thing in dotnet always confuses me... And as I don't often work with floating point types I always forget that a float is a single....
I wished they had gone with a more consistent naming, int32, in64, float32, float64....

Reply:
I really agree with you on this.

Reply:
solved via 41c5ee2 and a641f63 (push incoming)